libs/cardano-ledger-api/testlib/Test/Cardano/Ledger/Api/Arbitrary.hs

```diff
@@ -2,9 +2,12 @@
 
 module Test.Cardano.Ledger.Api.Arbitrary () where
 
-import Cardano.Ledger.Api.State.Query (MemberStatus)
+import Cardano.Ledger.Api.State.Query (MemberStatus, PState', mkPState')
 import Test.Cardano.Ledger.Common
 import Test.Cardano.Ledger.Dijkstra.Arbitrary ()
 
 instance Arbitrary MemberStatus where
   arbitrary = arbitraryBoundedEnum
+
+instance Arbitrary PState' where
+  arbitrary = mkPState' id <$> arbitrary
```

Comment:
Please don't make these kind of short cuts, since you could be unintentionally missing some cases:
```suggestion
  arbitrary = PState' <$> arbitrary <*> arbitrary <*> arbitrary <*> arbitrary
```